src/proposals/Proposals.jl

```diff
@@ -1,14 +1,13 @@
 """
     NestedSamplers.Proposals
-
 This module contains the different algorithms for proposing new points within a bounding volume in unit space.
-
```

Comment:
I'm not sure why you've removed this white space? Please revert this.

Reply:
I am sorry, I really don't remember when exactly did I do this! I will revert it.

Reply:
Done!